src/circuit/gadget/ecc/chip/mul_fixed.rs

```diff
@@ -557,6 +557,8 @@ impl ScalarFixed {
             .iter()
             .map(|window| {
                 if let Some(window) = window {
+                    // TODO: Remove this trait dependency
+                    use pasta_curves::arithmetic::SqrtRatio;
                     let window = window.get_lower_32() as usize;
```

Comment:
This dependency should not exist, but appears to have crept in because `FieldExt::get_lower_32` existed and was easy to use. We should find some other way to extract the window.